src/helperFunctions.js

```diff
@@ -310,7 +310,7 @@ class helpers {
         headers: headers
       })
         .then(async res => {
-          res.headers = new fetch.Headers(res.headers)
+          // res.headers = new fetch.Headers(res.headers)
```

Comment:
This causes some tests to fail, Why was this added @individual-it ?

Reply:
I think this is to have the headers as an object not array. What happens with that now? I guess we have an incompatibility between `xmlhttprequest` and `cross-fetch`

Reply:
this line was added in #636 but it seems not to be needed, tests pass at the stage of #636 without it and also pass now without it
CC @LukasHirt